lib/macho/structure.rb

```diff
@@ -15,8 +15,22 @@ def self.bytesize
 
     # @return [MachO::MachOStructure] a new MachOStructure initialized with `bin`
     # @api private
-    def self.new_from_bin(bin)
-      self.new(*bin.unpack(self::FORMAT))
+    def self.new_from_bin(endianness, bin)
```

Comment:
You might want to document the new `endianess` argument and the pre-existing but undocumented `bin` argument.